src/main/java/org/lisoft/lsml/view_fx/util/WeaponSummary.java

```diff
@@ -108,7 +108,7 @@ protected int computeValue() {
                 if (!weapons.isEmpty()) {
                     if (weapons.iterator().next() instanceof AmmoWeapon) {
                         return weapons.stream().map(w -> (AmmoWeapon) w)
-                                      .collect(Collectors.summingInt(AmmoWeapon::getAmmoPerPerShot));
+                                      .collect(Collectors.summingInt(AmmoWeapon::getRoundsPerShot));
```

Comment:
Here you can change to `getAmmoPerShot` directly as `w` is already of type `AmmoWeapon`.